cmds/exp/traceroute/main.go

```diff
@@ -41,6 +41,12 @@ func parseFlags(args []string) (*traceroute.Flags, error) {
 	f.BoolVar(&flags.TCP, "tcp", false, "Use TCP method. Same as -m tcp")
 	f.BoolVar(&flags.UDP, "udp", true, "Use UDP method. Same as -m udp")
 
+	// Check if args are supplied
+	if len(args[1:]) < 1 {
```

Comment:
what if the only args provided are a flag? won't f.Args still be zero length below? This test seems fragile.

Reply:
At line 48, you check len(leftoverArgs) > 1
why not just check if it is != 1. 
That will get the 0 args and more  than 1 args case. 
I think this PR needs a bit of a rethink.

Reply:
Thanks for the hints. Really appreciate it.